test/java/org/opendatakit/briefcase/export/Scenario.java

```diff
@@ -66,10 +66,6 @@ static Scenario nonGroup(DataType dataType) {
     return nonGroup("some-form", dataType, "field", null);
   }
 
-  static Scenario nonGroup(DataType dataType, String fieldName) {
```

Comment:
new/revisit

These test methods could be handy and also serve as an explanation of how to create those things. Also, since it's an internal test helper, not so important to leave it even if it's not used.